frontend/src/bundles/chat/components/chat-avatar/chat-avatar.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Text } from '~/bundles/common/components/components.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+import { MessageSender } from '../../enums/enums.js';
+
+type Properties = {
+    sender: ValueOf<typeof MessageSender>;
+    imageUrl?: string;
+    initials?: string;
+};
+
+const ChatAvatar: React.FC<Properties> = ({ sender, imageUrl, initials }) => {
+    return (
+        <Box
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            w="40px"
+            h="40px"
+            borderRadius="full"
+            bg={sender === MessageSender.USER ? 'blue.500' : 'green.500'}
+            color="white"
+            fontSize="md"
+            fontWeight="bold"
+        >
+            {imageUrl ? (
+                <img
+                    src={imageUrl}
+                    alt={initials}
+                    style={{
+                        borderRadius: '50%',
+                        width: '100%',
+                        height: '100%',
+                    }}
+                />
```

Comment:
Here you can use chakra's  <Box as="img" to avoid using html tags directly.

Reply:
@stefano-lacorazza Thanks